exercises/nucleotide-count/canonical-data.json

```diff
@@ -39,12 +63,28 @@
           }
         },
         {
-          "description": "strand with invalid nucleotides",
+          "description": "ignores case in a strand of mixed-case nucleotides.",
           "property": "nucleotideCounts",
-          "strand": "AGXXACT",
+          "strand": "aGcTtTtCaTtCtGaCtGcAaCgGgCaAtAtGtCtCtGtGtGgAtTaAaAaAaGaGtGtCtGaTaGcAgC",
```

Comment:
I'm not convinced it's necessary to add the extra functionality (of ignoring case), but I also don't care.